api/operator.go

```diff
@@ -304,3 +304,22 @@ func (op *Operator) LicenseGet(q *QueryOptions) (*LicenseReply, *QueryMeta, erro
 	}
 	return &reply, qm, nil
 }
+
+func (op *Operator) Metrics(q *QueryOptions) (string, error) {
```

Comment:
the `api/` package files usually follow the API endpoints, so `operator.go` contains endpoint methods to access `/v1/operator/` endpoints. Its a toss up since there will only be one `/v1/metrics` endpoint but I think it may make sense to just move it to a new `api/metrics.go` file

Reply:
On second thought, if the command is `nomad operator metrics` I think this is probably fine where it is

Reply:
This is one of a couple areas where I found competing standards between prior art and available documentation.  `v1/metrics` endpoint definitely isn't changing, but `nomad operator metrics` made more sense as the CLI command based on the "prefer subcommand over top level command" guidance from the [new cli command](https://github.com/hashicorp/nomad/blob/master/contributing/checklist-command.md) checklist.

In either case the overall goal was to make it easier for customers to gather metrics for support from specific targets.  Building the curl command in complex environments can be very tricky for inexperienced operators.